src/jitlayers.cpp

```diff
@@ -1665,17 +1659,6 @@ void fixupTM(TargetMachine &TM) {
     }
 }
 
-extern int jl_opaque_ptrs_set;
-void SetOpaquePointer(LLVMContext &ctx) {
-    if (jl_opaque_ptrs_set)
-        return;
-#ifndef JL_LLVM_OPAQUE_POINTERS
```

Comment:
If opaque pointers are always supposed to exist, then the definition of `JL_LLVM_OPAQUE_POINTERS` can be removed from https://github.com/JuliaLang/julia/blob/d298c44bc60eebdfc77b7f9c64ca913af1e5079d/src/llvm-version.h#L17-L19 this seems to be the only place where [the macro was used](https://github.com/search?q=repo%3AJuliaLang/julia%20JL_LLVM_OPAQUE_POINTERS&type=code).